_includes/tutorials/aggregating-sum/ksql/code/tutorial-steps/dev/transient-query.sql

```diff
@@ -0,0 +1,5 @@
+SELECT TITLE, 
+       SUM(TICKET_TOTAL_VALUE) AS TOTAL_VALUE
+FROM MOVIE_TICKET_SALES
+GROUP BY TITLE
```

Comment:
One other thing to briefly touch on in the narrative is the function that group by plays with getting the right results.

Reply:
I think I addressed that adequately in the next step. Your call.